### ISSUE TITLE
Lateralus and Tangents produces low volts at sample rates above 192kHz

### ISSUE THREAD

**Issue by stnslvk** [modlfo/VultModules#60]
Hi Leonardo

I've noticed Lateralus goes silent at sample rates above 192 kHz.
Tangents also share the same behavior.
Seems to be other Vult modules operates smoothly at those rates. Do not check every module yet. 

Please see video with a simple patch shows the possible issue.

```
Win 10
Rack standalone 2.2.1
Vult Free 2.0.9
```

thanks,
stas

https://user-images.githubusercontent.com/95583871/208322449-443ca6ad-c628-4fd1-a6ba-13c8dc0474d4.mp4

**Comment by modlfo** [comment]
Many modules only support the available sample rates that were available in Rack v1. Those are the sample rates available when those modules were released. I have to check them one by one and implement support for other sample rates.

**Comment by stnslvk** [comment]
Leonardo, thank you for looking into this. 
It is crucial for my sound production workflow to go to high sample rates at design stage to do some sort of FM, AM, cross modulation etc.
It will be very nice of you if you have time to adopt your lovely modules to work with high s/r.

**Comment by stnslvk** [comment]
Hi Leonardo.

Just installed [VultModulesFree-2.0.12-win-x64.vcvplugin](https://github.com/modlfo/VultModules/releases/download/v2.0.12/VultModulesFree-2.0.12-win-x64.vcvplugin)

‘Lateralus’ module still output low voltages for me when operating at sample rates exceeding 192kHz.
Same pre-requirement / steps / setup as per original ticket.

While the ‘Tangents’ module operates without any major issues, I have observed some peculiarities in its behavior. Specifically, when the ‘Cutoff’, ‘Resonance’, and ‘Drive’ are set to their minimum positions, and the ‘Drive’ knob is slowly turned, some non-linearities occur. This behavior may be by design, but I thought it was worth bringing to your attention.

The other filter modules are functioning as expected and produce excellent sound quality.

Thank you for your continued efforts in improving the Vult VCV plugs.

**Comment by modlfo** [comment]
I updated the binaries. Now Lateralus should work normal at 192kHz. 

The behavior of Tangents should be expected. The filter is nonlinear.

**Comment by stnslvk** [comment]
Leonardo, thanks a lot. 

I haven’t yet conducted a comprehensive analysis, but I have experimented with some crosspathing and audio rate modulation. The outcomes have been satisfactory so far.

**Comment by stnslvk** [comment]
I have encountered Tangents ‘nanV’ output twice today. However, I am still unable to pinpoint the exact root cause of this behavior.

Simple patch, one VCO is passed through Tangents, self-patching with the Cutoff, Res, and Drive inputs being modulated by module’s own output signal. The corresponding attenuverters were set to their maximum levels.

I used the standalone version of VCV Pro, with the engine frequency configured at 768 kHz running on Win 10.

Module initialization through the module menu does not help, only viable workaround to reset Tangents’s output appears to be a complete restart of the VCV rack itself.
